### PR TITLE
Introduce GH actions for version tag check and release notes presence

### LINKED ISSUE #70: Simplify GH Workflows using new GH Actions
## Background
Current GH workflows for Release notes checking and version tag checking can be replace by 3rd party GH actions.

## Feature
Replace them.
Use these actions:
- https://github.com/AbsaOSS/version-tag-check/releases/tag/v0.1.0 in release_draft.yml
- https://github.com/AbsaOSS/release-notes-presence-check/releases/tag/v0.1.0 in check_pr_release_notes.yml

### PR DESCRIPTION
**Pull request by MobiTikula**
Release Notes:
- Switched to use new GH action. Simplified workflow files.

Closes #70

### CODE REVIEW

File: .github/workflows/release_notes_comments_migration.yml [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
 name: Copy Release Notes to Related Issues
 
 on:
```
Comment (miroslavpojer) [inline .github/workflows/release_notes_comments_migration.yml:19]: This file can be deleted as it si not used anymore.

**Review by miroslavpojer — COMMENTED** [review]
- [ ] code review

File: .github/workflows/release_draft.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-# GH workflow example from https://github.com/AbsaOSS/generate-release-notes/blob/master/examples/release_draft.yml
+# GH workflow source https://github.com/AbsaOSS/generate-release-notes
```
Comment (miroslavpojer) [inline .github/workflows/release_draft.yml:17]:
```suggestion
# GH workflow source https://github.com/AbsaOSS/generate-release-notes/blob/master/.github/workflows/release_draft.yml
```

File: .github/workflows/check_pr_release_notes.yml [hunk 3]
```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-# GH workflow example from https://github.com/AbsaOSS/generate-release-notes/blob/master/examples/check_pr_release_notes.yml
+# GH workflow source https://github.com/AbsaOSS/release-notes-presence-check
```
Comment (miroslavpojer) [inline .github/workflows/check_pr_release_notes.yml:17]:
```suggestion
# GH workflow source https://github.com/AbsaOSS/generate-release-notes/blob/master/.github/workflows/check_pr_release_notes.yml
```

Comment (MobiTikula) [reply .github/workflows/check_pr_release_notes.yml]: [92108ad](https://github.com/AbsaOSS/living-doc-generator/pull/71/commits/92108ad1a61ad4f188f7a768eef9223f014d7f5e)

File: .github/workflows/release_draft.yml (hunk 2 above)
Comment (MobiTikula) [reply .github/workflows/release_draft.yml]: [92108ad](https://github.com/AbsaOSS/living-doc-generator/pull/71/commits/92108ad1a61ad4f188f7a768eef9223f014d7f5e)

**Review by miroslavpojer — APPROVED** [review]

**Pull request merged**